Sources/AWSLambdaRuntimeCore/LambdaRunner.swift

```diff
@@ -78,7 +78,13 @@ internal final class LambdaRunner {
                 allocator: self.allocator,
                 invocation: invocation
             )
-            logger.debug("sending invocation to lambda handler")
+            // when log level is trace or lower, print the first Kb of the payload
+            if logger.logLevel <= .trace, let buffer = bytes.getSlice(at: 0, length: max(bytes.readableBytes, 1024)) {
+                logger.trace("sending invocation to lambda handler",
+                             metadata: ["1024 first bytes": .string(String(buffer: buffer))])
```

Comment:
I suspect `String(buffer: buffer)` may fail if buffer is not a string, please make sure it does not crash

Reply:
According to the AWS Lambda service API documentation, the `payload` is always a string
https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html